web/packages/teleport/src/services/audit/makeEvent.ts

```diff
@@ -2444,6 +2444,27 @@ export const formatters: Formatters = {
       return `App authentication using [${app_auth_config}] failed: ${error}`;
     },
   },
+  [eventCodes.VNET_CONFIG_CREATE]: {
+    type: 'vnet_config.create',
+    desc: 'VNet config created',
+    format: ({ user, name }) => {
+      return `User [${user}] created the VNet config [${name}]`;
+    },
+  },
+  [eventCodes.VNET_CONFIG_UPDATE]: {
+    type: 'vnet_config.update',
+    desc: 'VNet config updated',
+    format: ({ user, name }) => {
+      return `User [${user}] updated the VNet config [${name}]`;
+    },
+  },
+  [eventCodes.VNET_CONFIG_DELETE]: {
+    type: 'vnet_config.delete',
+    desc: 'VNet config deleted',
+    format: ({ user, name }) => {
+      return `User [${user}] deleted the VNet config [${name}]`;
```

Comment:
since the name is a constant (this is a singleton resource) I think we can omit it, act like cluster_networking_config or session_recording_config